compatibility/sandbox-migration/runner/Migration/Runner.hs

```diff
@@ -67,27 +67,37 @@ main = do
                 , "--host=localhost", "--port=" <> show p
                 ]
         hPutStrLn stderr "<-- Uploaded model DAR"
-        runTest jdbcUrl platformAssistants (ProposeAccept.test step modelDar)
+        runTest "propose-accept" jdbcUrl platformAssistants (ProposeAccept.test step modelDar)
+    withPostgres $ \jdbcUrl -> do
```

Comment:
If we do start postgres separately for each test we might as well move it into `runTest`.

Reply:
Yeah, I faced some issues initially because I was trying to run starting from the earliest version after I completed a step-by-step test over the entire version range. This caused an older Ledger API server to use a newer underlying database schema, causing weird runtime issues. Ideally it would be nice to have a way of using a single Postgres instance and running all steps for all tests for a given version before moving on. If you think this is feasible but perhaps requires a bit of tinkering around, I'd be ok moving the `withPostgres` inside `runTest` and use the new approach as part of a separate contribution.

Reply:
Should be doable but it’s a bit of work so I would suggest to move `withPostgres` inside the test for now and address this separately.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/6298/commits/472ae2786225c9408d53ff0de86917e1df0cae64